docs/how-to/managed-data/upload-data.md

```diff
@@ -85,12 +86,12 @@ This page describes how to upload data to Deep Origin Data.
     We can upload a file using 
 
     ```py
-    from deeporigin.managed_data import _api, api
-    _api.upload_file("/path/to/file.fasta")
+    from deeporigin.managed_data import api
+    api.upload_file("/path/to/file.fasta")
     ```
 
     This uploads files to the "Staging Area" of Deep Origin
-    Data, and returns data that conforms to [DescribeFileResponse][src.managed_data.schema.DescribeFileResponse]. An example is shown below:
```

Comment:
unfortunately these models have gone away and i haven't found an easy way to include the models in the generated code back here